src/test/resources/org/opensearch/agent/tools/register_text_embedding_model_request_body.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "huggingface/sentence-transformers/all-MiniLM-L12-v2",
```

Comment:
Since we'll register model for each test case, all-MiniLM-L12-v2 is too large for integ test here. We can use a tiny mocked embedding model in ml-commons. ref: https://github.com/opensearch-project/neural-search/blob/c198298531109af63c8d9d493529cdb913e9cce3/src/test/resources/processor/UploadModelRequestBody.json#L14

Reply:
used this traced_small_model model, also works well. switch to traced_small_model